COMMUNICATON.md

```diff
@@ -0,0 +1,33 @@
+# Communication
+
+## Code of conduct
+
+This project has adopted the [GitHub community guidelines](https://help.github.com/en/github/site-policy/github-community-guidelines).
+
+## Issues
+
+Issues are used as the primary method for tracking anything to do with the Fairlearn project.
+
+See the [README](https://github.com/fairlearn/fairlearn/blob/master/CONTRIBUTING.md#issues)
+for information on how to file an issue if you think you have found a bug or have a feature request.
+
+## Proposals
+
+Design proposals for larger changes go into the [fairlearn-proposals repository](https://github.com/fairlearn/fairlearn-proposals). If you want to submit a proposal, just create a pull request there and maintainers will review it.
+
+## Microsoft Teams
+
+Most real time discussion and anything that wouldn't fit into the above categories happens in [Microsoft Teams](https://teams.microsoft.com/l/team/19%3a441d98d202554f25a4f3030a8c27d3c0%40thread.skype/conversations?groupId=70d3c25a-81db-4a52-85d2-5b56e50fe985&tenantId=b6185859-4461-4dca-b558-0cdcd345d3ac). Feel free to join!
+
+## Meetings
+
+We are in the process of setting up regular contributor meetings. Stay tuned!
+
+We have a TBD development meeting every TBD at TBD US Pacific Time and all are welcome.
+
+We keep notes from each meeting in [Microsoft Teams](https://teams.microsoft.com/l/team/19%3a441d98d202554f25a4f3030a8c27d3c0%40thread.skype/conversations?groupId=70d3c25a-81db-4a52-85d2-5b56e50fe985&tenantId=b6185859-4461-4dca-b558-0cdcd345d3ac).
```

Comment:
The notes will be much better kept in an `administrative` repo or something, rather than a message lost in the conversation history.

Reply:
Good point. Since these meetings haven't taken place this is also a change I can make without any pain :-)

Reply:
If you check the PR in the community repo there's now a directory for these notes!